client/src/App.jsx

```diff
@@ -9,7 +9,7 @@ import RequestBouquet from './components/RequestBouquet';
 import NavBar from './components/NavBar';
 import EntryForm from './components/EntryForm';
 import Login from './components/Login';
-import './App.css'
+// import './App.css' //not sure I need this line of code??
```

Comment:
If you don't need the styling, you can remove it.

Reply:
Thank you Philip!